packages/framework/src/Model/Product/Search/FilterQueryFactory.php

```diff
@@ -14,4 +66,152 @@ public function create(string $indexName): FilterQuery
     {
         return new FilterQuery($indexName);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterData $productFilterData
+     * @param string $orderingModeId
+     * @param int $page
+     * @param int $limit
+     * @param int $categoryId
+     * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
+     */
+    public function createListableProductsInCategoryFilterQuery(
+        ProductFilterData $productFilterData,
+        string $orderingModeId,
+        int $page,
+        int $limit,
+        int $categoryId
+    ): FilterQuery {
+        return $this->createFilterQueryWithProductFilterData($productFilterData, $orderingModeId, $page, $limit)
+            ->filterByCategory([$categoryId]);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterData $productFilterData
+     * @param string $orderingModeId
+     * @param int $page
+     * @param int $limit
+     * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
+     */
+    public function createFilterQueryWithProductFilterData(
```

Comment:
It would be nice to unify method names (see `createFilterQueryWithProductFilterData` Vs. `createListableProductsInCategoryFilterQuery`) and `FilterQuery` in method names seems redundant to me

Reply:
fixed in 7719ac882bdbf41e88e7853c8c8d5ed2f2644273